Hearthstone Deck Tracker/Controls/DeckImportingControl.xaml

```diff
@@ -26,15 +26,16 @@
             <Border BorderThickness="1" BorderBrush="{DynamicResource AccentColorBrush}" HorizontalAlignment="Center" Margin="5">
                 <DockPanel Margin="0,0,0,5">
                     <StackPanel Orientation="Horizontal" DockPanel.Dock="Top">
+                        <CheckBox Name="ImportAllChk" IsChecked="True" Margin="5,0,0,0" Unchecked="onImportAll_Modified" Checked="onImportAll_Modified"/>

```

Comment:
For consistency sake, the name should be `CheckBoxImportAll` and the events `CheckBoxImportAll_OnModified`.